app/models/request_issues_update.rb

```
@@ -50,7 +50,12 @@ def establish!
 
     potential_end_products_to_remove = []
     removed_issues.select(&:end_product_establishment).each do |request_issue|
-      request_issue.end_product_establishment.remove_contention!(request_issue)
+      begin
+        request_issue.end_product_establishment.remove_contention!(request_issue)
+      rescue EndProductEstablishment::ContentionNotFound => _err
```

alternately, we could report the error to Sentry if we wanted to track the condition.